profiler/profiler.go

```diff
@@ -22,25 +22,23 @@ const outChannelSize = 5
 var (
 	mu             sync.Mutex
 	activeProfiler *profiler
+	containerID    = internal.ContainerID()
```

Comment:
Let's remove this.

Reply:
This is useful for the testing of the addition of the container ID header.

Reply:
True. In that case, can we add a comment?
```suggestion
	containerID    = internal.ContainerID() // replaced in tests
```